modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js

```diff
@@ -615,20 +619,33 @@ return view.extend({
 			uci.unset('dhcp', section, 'dns');
 		};
 
-		so = ss.option(form.Value, 'mac', _('MAC address'));
-		so.datatype = 'list(macaddr)';
+		so = ss.option(form.Value, 'mac',
+			_('MAC address(es)'),
+			_('The hardware address(es) of this entry/host, separated by spaces.'));
+		so.validate = function(section_id, value) {
+			var macaddrs = L.toArray(value);
+
+			for (var i = 0; i < macaddrs.length; i++)
+				if (!macaddrs[i].match(/^([a-fA-F0-9]{2}|\*):([a-fA-F0-9]{2}:|\*:){4}(?:[a-fA-F0-9]{2}|\*)$/))
+				return _('Expecting a valid MAC address, optionally including wildcards');
+
+			return true;
+		};
 		so.rmempty  = true;
 		so.cfgvalue = function(section) {
 			var macs = L.toArray(uci.get('dhcp', section, 'mac')),
-			    result = [];
+				result = [];
 
 			for (var i = 0, mac; (mac = macs[i]) != null; i++)
```

Comment:
Why do we not use the forEach pattern https://www.w3schools.com/jsref/jsref_forEach.asp?
Or
Use the iteration like in https://github.com/openwrt/luci/pull/5401/files#diff-90caf036d2316d858be5df6b382c44d4130a66c52cc7b6fda8ee715971a4f9bdR628

Reply:
Not sure - it's ES5 - is that what OWRT 21.02 aims for?